packages/genesys-spark-components/src/components/stable/gux-listbox-multi/gux-listbox-multi.tsx

```diff
@@ -256,9 +257,9 @@ export class GuxListboxMulti {
       listboxOption.selected = this.getSelectedValues().includes(
         listboxOption.value
       );
-      if (this.filterType !== 'custom') {
-        listboxOption.filtered = !listboxOption.textContent
-          .trim()
+      if (this.filterType !== 'custom' && this.filterType !== 'none') {
```

Comment:
Looks like my change exposed this missing condition and that's why the snapshots and gux-phone-input were acting weird